src/Database/DBA.php

```diff
@@ -82,29 +82,29 @@ public static function connect($serveraddr, $user, $pass, $db, $charset = null)
 			}
 
 			try {
-				self::$db = @new PDO($connect, $user, $pass);
-				self::$db->setAttribute(PDO::ATTR_EMULATE_PREPARES, false);
+				self::$connection = @new PDO($connect, $user, $pass);
+				self::$connection->setAttribute(PDO::ATTR_EMULATE_PREPARES, false);
 				self::$connected = true;
 			} catch (PDOException $e) {
 			}
 		}
 
 		if (!self::$connected && class_exists('mysqli')) {
 			self::$driver = 'mysqli';
-			self::$db = @new mysqli($server, $user, $pass, $db, $port);
+			self::$connection = @new mysqli($server, $user, $pass, $db, $port);
```

Comment:
This change will cause conflict with my fix for missing `$port` variable, I guess?

Reply:
Definitely.

Reply:
Then what should be merged first? Mine or this?

Reply:
Either, it just means the other will have to be updated afterwards.